Numerge.Gui/ViewLocator.cs

```diff
@@ -0,0 +1,35 @@
+// Copyright (c) The Avalonia Project. All rights reserved.
+// Licensed under the MIT license. See licence.md file in the project root for full license information.
+
+using System;
+using Avalonia.Controls;
+using Avalonia.Controls.Templates;
+using Numerge.Gui.ViewModels;
+
+namespace Numerge.Gui
+{
+    public class ViewLocator : IDataTemplate
```

Comment:
We are assigning the `DataContext` near the entry point, so probably worth removing the `ViewLocator` we don't use.